releases/release-1.28/release-notes/maps/pr-117120-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 117120
+releasenote:
+  text: Bumped metrics-server to v0.6.3.
```

Comment:
```suggestion
  text: Bumped metrics-server to `v0.6.3`.
```

Reply:
The Docs [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) do not use backticks for version numbers, should we use them in release notes? No problem in changing it, but would like more input on it.

Reply:
Woots! And for context, I think I learned the backticks from @ramrodo. I think they read nicer on Release Notes with them, but maybe we should document it somewhere for the release notes team?